code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -883,37 +883,37 @@
 
 /datum/chemical_reaction/foresttea1
 	name = "Forest Tea"
-	id = /datum/reagent/consumable/tea/forest
+	id = "forest_tea_fuzz"
```

Comment:
I thought IDs were deprecated in that manner?
It's not used elsewhere, at least, unless I'm mistaken.

Reply:
No. It's not deprecated, it's a mistake. IDs need to be unique; only Citadel chems use path IDs. TG chems use string IDs.